addon/components/single-event.js

```diff
@@ -1,32 +1,36 @@
-import { inject as service } from '@ember/service';
 import Component from '@ember/component';
 import { computed } from '@ember/object';
+import { inject as service } from '@ember/service';
 import { isBlank, isEmpty } from '@ember/utils';
 import layout from '../templates/components/single-event';
 import moment from 'moment';
 
 export default Component.extend({
-  store: service(),
   intl: service(),
   router: service(),
-  layout,
-  event: null,
+  store: service(),
+
   classNames: ['single-event'],
+
   'data-test-single-event': true,
+  event: null,
+  layout,
 
-  taughtBy: computed('intl.locale', 'event.instructors', function(){
-    const instructors = this.get('event.instructors');
-    if (isEmpty(instructors)) {
-      return '';
+  taughtBy: computed('event.instructors', 'intl.locale', {
+    get() {
+      const instructors = this.get('event.instructors');
+      if (isEmpty(instructors)) return '';
```

Comment:
I'll add linting to prevent this ASAP sorry, but single line if without curlys aren't our pattern so this should be changed to ```
if (isEmpty(instructors)) {
  return '';
}
```

Reply:
Darn. I really love single line `if` :(